modules/oe_webtools_laco_widget/src/Form/WebtoolsLacoWidgetSettingsForm.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_webtools_laco_widget\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Settings form for module.
+ */
+class WebtoolsLacoWidgetSettingsForm extends ConfigFormBase {
+
+  /**
+   * Name of the config being edited.
+   */
+  const CONFIGNAME = 'oe_webtools_laco_widget.settings';
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $form['intro'] = [
+      '#type' => 'item',
+      '#description' => $this->t('For more information check the module <a href="@href">documentation</a>.', ['@href' => 'https://github.com/openeuropa/oe_webtools/blob/master/modules/oe_webtools_laco_widget/README.md']),
+    ];
+    $form['include'] = [
+      '#type' => 'textarea',
+      '#title' => $this->t('Include'),
+      '#default_value' => implode(PHP_EOL, $this->config(static::CONFIGNAME)->get('include')),
+      '#description' => $this->t('CSS selectors within which to apply the widget.'),
+    ];
+    $form['exclude'] = [
+      '#type' => 'textarea',
+      '#title' => $this->t('Exclude'),
+      '#default_value' => implode(PHP_EOL, $this->config(static::CONFIGNAME)->get('exclude')),
+      '#description' => $this->t('CSS selectors to exclude the widget.'),
+    ];
+    $form['coverage'] = [
+      '#type' => 'fieldset',
+      '#title' => $this->t('Coverage'),
+    ];
+    $form['coverage']['coverage_document'] = [
+      '#type' => 'select',
+      '#title' => $this->t('Document'),
+      '#options' => ['any' => 'any', 'other' => 'other', 'false' => 'false'],
```

Comment:
Shouldn't we provide this options as a constant or as a static variable?

Reply:
We're not making any use of these options, so for me there's no advantage in having constants for this. I'm not against, I just don't see the propose.